cipd_packages/device_doctor/lib/src/android_device.dart

```diff
@@ -420,7 +424,7 @@ class AndroidDevice implements Device {
 
     // Listen to the stdout and stderr streams
     LineSplitter().convert(result.stdout).forEach((data) {
-      final packageMatch = RegExp(r'package:(.+)$').firstMatch(data);
+      final packageMatch = RegExp(r'package:.*=(.+)$').firstMatch(data);
```

Comment:
Based on go/flutter-luci-playbook#android-install-failed-insufficient-storage
We want to uninstall only those flutter apps:
```
adb shell pm list packages -f | grep '/data/app/com.example\|/data/app/com.yourcompany\|flutter' | awk '{split($0,a,"="); print a[2]}' | xargs -I % adb shell pm uninstall %
```

Reply:
Here are what it found on one of our devices:

```
$ adb shell pm list packages -3
package:com.google.android.keep
package:com.google.android.apps.translate
package:io.flutter.examples.hello_world
package:com.google.android.apps.docs.editors.docs
package:com.google.android.apps.tachyon
package:com.google.android.apps.magazines
package:com.google.android.apps.walletnfcrel
package:com.google.android.apps.chromecast.app
package:com.google.android.apps.podcasts
package:com.google.android.apps.subscriptions.red
```

Reply:
As mentioned above by @godofredoc , this list seems contains some system packages and may cause crash. Can we try adding the grep to give it a shot?